server/server.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const express = require('express');
+const path = require('path');
+
+const app = express();
+const { PORT = 3000 } = process.env;
+const distFolder = `${__dirname}/dist`;
+
+app.use(express.static(distFolder))
+  .use((req, res) => {
+    res.sendFile(path.resolve(`${distFolder}/index.html`));
+  });
```

Comment:
Когда есть чейнинг и методы не умещаются на одну строку, то писать стоит так:

```suggestion
app
  .use(express.static(distFolder))
  .use((req, res) => {
    res.sendFile(path.resolve(`${distFolder}/index.html`));
  });
```

Проглядел(

Reply:
Поправила, спасибо